workshop/README.md

```diff
@@ -250,17 +345,16 @@ The payment transaction accepts one parameter in the `asset.data` field (send by
 ```
 
 ### Implementation Details
-- Validate if invoice ID exists and if the client is sending the required balance to the freelancer
-- Make sure the tokens are send to the right account
-- We do not keep track on the payment status (in the account asset field) as we can query the API to find all payment transactions and look for the invoiceID.
+- Validate if invoice ID exists and if the client is sending the required balance to the freelancer (Tip: dynamic)
+- Make sure the tokens are sent to the right account
 
 ### Exploring Payment Transaction
-If you haven't done this step for the invoice transaction: navigate in your terminal to the `./transactions` folder. Run `npm install` to again install the required dependencies as we treat the `/transactions` folder as a separate module.
+If you haven't installed the dependencies inside the `./transactions` folder yet, navigate in your terminal to the `./transactions` folder. Run `npm install` to again install the required dependencies as we treat the `/transactions` folder as a separate module.
 
-Now, let's get technical. Open the file at `./transactions/payment_transaction.js`. For this transaction, most code needs to be written. Good luck!
+Now, let's get technical. Open the file at `./transactions/payment_transaction.js`. For this transaction type (14), most code needs to be written. Good luck!
 
 ### Exploring Payment Transaction
-The idea for the `PaymentTransaction` is that you write the logic yourself based on the requirements and tips I give. The payment transaction will extend from the `TransferTransaction` although we do not recommend this (it simplifies things).
+The idea for the `PaymentTransaction` is that you write the logic yourself based on the requirements and tips we give to you. The payment transaction will extend from the `TransferTransaction` although we do not recommend this (it simplifies things).
```

Comment:
We are not very consistent on this. Though we do not recommend to extend from existing tx types, we do that in Cashback and Lisk Bills to simplify things.

I think it's bad to introduce best practice and then immediatley break with it in your own tutorial. Either implement the tx type from Basetransaction or drop the comment here imho.

Reply:
Will drop the comment, indeed, need to address this to be consistent.